resources/zib/zib-LaboratoryTestResult.xml

```diff
@@ -150,7 +148,7 @@
       <alias value="Onderzoek" />
       <alias value="TestCode" />
       <binding>
-        <strength value="required" />

```

Comment:
This will be a deviation from the zib. Is that justified?
Also, the profile currently already allows codes from NHG and LOINC (incl NL Labcodeset). So given NICTIZ-17167 (which is a private board) I don't see a reason to adjust this.